panic: inconsistent label cardinality: expected 21 label values but got 20

**Describe the bug**
A clear and concise description of what the bug is.

follow https://github.com/sustainable-computing-io/kepler

after `~/kube-prometheus# kubectl apply -f manifests/`, saw this error

seems something inconsistent ..
```
panic: inconsistent label cardinality: expected 21 label values but got 20 in []string{"system_processes", "system", "containerd", "388246", "2674068", "0", "0", "0", "0", "0", "0", "3", "151", "17428480", "1070764032", "0", "0", "0", "0", "0"}

goroutine 101 [running]:
github.com/prometheus/client_golang/prometheus.MustNewConstMetric(...)
        /opt/app-root/src/github.com/sustainable-computing-io/kepler/vendor/github.com/prometheus/client_golang/prometheus/value.go:107
github.com/sustainable-computing-io/kepler/pkg/collector.(*Collector).Collect(0xc000400710, 0xc0000fdf60?)
        /opt/app-root/src/github.com/sustainable-computing-io/kepler/pkg/collector/collector.go:315 +0x1a76
github.com/prometheus/client_golang/prometheus.(*Registry).Gather.func1()
        /opt/app-root/src/github.com/sustainable-computing-io/kepler/vendor/github.com/prometheus/client_golang/prometheus/registry.go:446 +0xfb
created by github.com/prometheus/client_golang/prometheus.(*Registry).Gather
        /opt/app-root/src/github.com/sustainable-computing-io/kepler/vendor/github.com/prometheus/client_golang/prometheus/registry.go:538 +0xb0b

```



**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - OS: [e.g. iOS]
 - Browser [e.g. chrome, safari]
 - Version [e.g. 22]

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6]
 - OS: [e.g. iOS8.1]
 - Browser [e.g. stock browser, safari]
 - Version [e.g. 22]

**Additional context**
Add any other context about the problem here.

--- a/pkg/collector/collector.go
+++ b/pkg/collector/collector.go
@@ -56,8 +56,8 @@ var (
 	basicPodLabels []string = []string{
 		"pod_name", "pod_namespace", "command",
 	}
-	prometheusMetrics []string = getPrometheusMetrics()
-	podEnergyLabels   []string = append(basicPodLabels, prometheusMetrics...)
+	prometheusMetrics []string
+	podEnergyLabels   []string
 	basicNodeLabels   []string = []string{
 		"node_name", "cpu_architecture",
 	}
@@ -76,6 +76,11 @@ type Collector struct {
 	modules  *attacher.BpfModuleTables
 }
 
+func setPodStatProm() {
+	prometheusMetrics = getPrometheusMetrics()
+	podEnergyLabels = append(basicPodLabels, prometheusMetrics...)
+}
+
 func New() (*Collector, error) {
 	c := &Collector{}
 	c.setNodeDescriptionGroup()
@@ -84,6 +89,7 @@ func New() (*Collector, error) {
 
 func (c *Collector) Attach() error {
 	m, err := attacher.AttachBPFAssets()
+	setPodStatProm()
 	if err != nil {
 		return fmt.Errorf("failed to attach bpf assets: %v", err)
 	}

--- a/pkg/collector/collector_test.go
+++ b/pkg/collector/collector_test.go
@@ -52,6 +52,7 @@ func convertPromToValue(body []byte, metric string) (int, error) {
 var _ = Describe("Test Collector Unit", func() {
 	It("Init and Run", func() {
 		newCollector, err := New()
+		setPodStatProm()
 		Expect(err).NotTo(HaveOccurred())
 		err = prometheus.Register(newCollector)
 		Expect(err).NotTo(HaveOccurred())

--- a/pkg/collector/metrics_test.go
+++ b/pkg/collector/metrics_test.go
@@ -8,6 +8,7 @@ import (
 
 var _ = Describe("Test Metric Unit", func() {
 	It("Check feature values", func() {
+		setPodStatProm()
 		Expect(len(uintFeatures)).Should(BeNumerically(">", 0))
 		Expect(len(podEnergyLabels)).Should(BeNumerically(">", 0))
 		Expect(len(podEnergyLabels)).Should(BeNumerically(">", 0))
@@ -15,6 +16,7 @@ var _ = Describe("Test Metric Unit", func() {
 	})
 
 	It("Check convert values", func() {
+		setPodStatProm()
 		v := NewPodEnergy("podA", "default")
 		v.EnergyInCore = &UInt64Stat{
 			Curr: 10,

